Taxonomy Manager: make Elipsis Menu Click Target Larger.

timmyc opened the pull request:
Closes https://github.com/Automattic/wp-calypso/projects/11#card-836137

From p5j4vm-1pD-p2, @alisterscott noted that the click target for the ellipsis menu in the TaxonomyManager is quite small, to the point where you often accidentally click on the row behind and the edit window opens.

This branch wraps the gridicon in an element to make the target bigger.

__To Test__
- Visit a [site settings page](http://calypso.localhost:3000/settings/writing), click 'Categories'
- On the subsequent page, interact with the action menu for a category

/cc @youknowriad

alisterscott commented:
Thanks for the quick fix @timmyc 

I tested this out, and the opening of the menu is much better - thanks.

However, closing the menu doesn't match, so clicking the same place (above the elipsis) twice opens but doesn't close the menu.

See: 
![open close](https://cloud.githubusercontent.com/assets/128826/20418896/7891b120-ad9d-11e6-979d-eccc1310a40e.gif)

youknowriad commented on client/blocks/taxonomy-manager/list-item.jsx:
```diff
@@ -58,14 +58,16 @@ class TaxonomyManagerListItem extends Component {
+							'taxonomy-manager__list-item-toggle': true,
+							'is-active': this.state.popoverMenuOpen
+						} ) }
+						onClick={ this.togglePopoverMenu }
```
shouldn't we drop the onClick here ?

timmyc commented:
Rebased and fixed the above issue - the cause there was the `onClose` on the `PopoverMenu` was re-opening the menu.  The fix was to move the `ref` to the wrapper, which moves the menu to the bottom of the wrapper element, but I suppose that makes sense since the menu should be tied to the element that toggles it.

youknowriad commented:
LGTM 🚢

timmyc commented:
@youknowriad thanks - I'll let you merge in #9508 and rebase this.
